EXT_BeamPhysics_PRAYTRACE.md

```diff
@@ -0,0 +1,171 @@
+Additional definitions for photon raytracing for BeamPhysics Extension of the openPMD Standard
+=========================================================================================================
+openPMD extension name: `Raytracing`
```

Comment:
Thanks a lot!
Can you please cross-link this extension in `STANDARD.md`? :)
Here is the corresponding location:
https://github.com/openPMD/openPMD-standard/blob/upcoming-2.0.0/STANDARD.md#domain-specific-extensions

Reply:
I pushed a new commit to add the Raytracing extension to the list of domain specific extensions